lib/src/linden/xcolors.dart

```diff
@@ -21,7 +21,7 @@ class XColors {
   /// Private Colors never use directly
   // Primary colors
   static const _blue = Color(0xff233564);
-  static const _black = Color(0xff131313);
+  static const _black = Color(0xff0F1A18);
```

Comment:
💅 ideally we should work with `NewXColorsx`.
There is a list of all available colors, which we can use (copied from figma).
Again, ideally we should stop using this file and remove it complitely.

Reply:
I understand, but it's not part of this story/PR. It's a separate story that Peter is working on. Check this for ref: https://github.com/xaynetwork/xayn_design/pull/69#discussion_r810068497

Reply:
oka, thanks 🙏